core/bin/external_node/src/main.rs

```diff
@@ -267,7 +268,11 @@ async fn init_tasks(
         .context("failed to build a tree_pool")?;
     let tree_handle = task::spawn(metadata_calculator.run(tree_pool, tree_stop_receiver));
 
-    let consistency_checker_handle = tokio::spawn(consistency_checker.run(stop_receiver.clone()));
+    let l1_batch_commit_data_generator = Arc::new(RollupModeL1BatchCommitDataGenerator {});
```

Comment:
I think you should make sure that EN can work as well. EN verifies the pubdata sent to L1, so if main node is in validium mode, EN (in its current form) will probably not work.